src/main/java/de/azapps/kafkabackup/common/topic/restore/RestoreTopicsArgsWrapper.java

```diff
@@ -0,0 +1,82 @@
+package de.azapps.kafkabackup.common.topic.restore;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Properties;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public class RestoreTopicsArgsWrapper {
+
+  public static final String KAFKA_CONFIG_BACKUP_BUCKET = "aws.s3.bucketNameForConfig";
+  public static final String KAFKA_BOOTSTRAP_SERVERS = "kafka.bootstrapServers";
+
+  public static final String RESTORE_DRY_RUN = "restore.dryRun";
+  public static final String RESTORE_IGNORE_EXISTING_TOPICS = "restore.ignoreExistingTopics";
+  public static final String RESTORE_TIME = "restore.time";
```

Comment:
I guess this means we will try to find the latest config with updated_at <= restore.time?

Reply:
Exactly. Could have pointed that out.